test/functionalTest/cases/4330_onlyChangedAttrs_built_in_metadata/onlychangedattrs_not_working_with_built_in_metadata.test

```diff
@@ -0,0 +1,226 @@
+sleep 2.5s
+payload='{
+   "A": {
+     "value": "zzz",
+     "type": "Text"
+   }
+}'
+orionCurl --url /v2/entities/E1/attrs --payload "$payload" --tenant tk --servicePath /
+echo
+echo
+
+
+echo '05. Dump accumulator and see only A atribute in both notifications'
+echo '=================================================================='
+accumulatorDump
+echo
+echo
+
+
+--REGEXPECT--
+01. Create entity E1 with attributes A and B
+============================================
+HTTP/1.1 201 Created
+Date: REGEX(.*)
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Location: /v2/entities/E1?type=T
+Content-Length: 0
+
+
+
+02. Create subscription with onlyChangedAttrs true covering the entity
+======================================================================
+HTTP/1.1 201 Created
+Date: REGEX(.*)
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Content-Length: 0
+
+
+
+03. Update E1-A
+===============
+HTTP/1.1 204 No Content
+Date: REGEX(.*)
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+
+
+
+04. Sleep until cache refres, update E1-A again
+===============================================
+HTTP/1.1 204 No Content
+Date: REGEX(.*)
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+
+
+
+05. Dump accumulator and see only A atribute in both notifications
+==================================================================
+POST http://localhost:9997/notify
+Fiware-Servicepath: /
+Content-Length: 170
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Ngsiv2-Attrsformat: normalized
+Host: localhost:REGEX(\d+)
+Accept: application/json
+Fiware-Service: tk
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36}); cbnotif=1
+
+{
+    "data": [
+        {
+            "A": {
+                "metadata": {
+                    "previousValue": {
+                        "type": "Text",
+                        "value": "foo"
+                    }
+                },
+                "type": "Text",
+                "value": "bar"
+            },
+            "id": "E1",
+            "type": "T"
+        }
+    ],
+    "subscriptionId": "REGEX([0-9a-f]{24})"
+}
+=======================================
+POST http://localhost:9997/notify
+Fiware-Servicepath: /
+Content-Length: 170
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Ngsiv2-Attrsformat: normalized
+Host: localhost:REGEX(\d+)
+Accept: application/json
+Fiware-Service: tk
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36}); cbnotif=1
+
+{
+    "data": [
+        {
+            "A": {
+                "metadata": {
+                    "previousValue": {
+                        "type": "Text",
+                        "value": "bar"
+                    }
+                },
+                "type": "Text",
+                "value": "zzz"
+            },
+            "id": "E1",
+            "type": "T"
+        }
+    ],
+    "subscriptionId": "REGEX([0-9a-f]{24})"
+}
```

Comment:
The test expectations includes attribute `A` and `previousValue` metadata

Reply:
Great! :)

NTC